Cesium3DTilesSelection/include/Cesium3DTilesSelection/RasterOverlay.h

```diff
@@ -86,6 +86,15 @@ struct CESIUM3DTILESSELECTION_API RasterOverlayOptions {
    * @brief Whether or not to display the credits on screen.
    */
   bool showCreditsOnScreen = false;
+
+  /**
+   * @brief Arbitrary data that will be passed to {@link prepareRasterInLoadThread},
+   * for example, data to control the per-raster overlay client-specific texture
+   * properties.
+   *
+   * This pointer can and will be accessed by multiple threads simultaneously.
+   */
+  void* pRendererOptions = nullptr;
```

Comment:
Sorry I didn't think to mention this earlier, but how about using `std::any` instead of `void*`? I think that would make it easier to control the object's lifetime, as well as being generally safer.